<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <rosparam command="load" file="$(find ssd_ros)/cfg/ssd_mobilenet_v1_coco.yaml" ns="ssd_infer"/>
  <node pkg="ssd_ros" name="ssd_infer" type="ssd_node" output="screen">
    <remap from="/ssd_infer/input_topic" to="/camera/color/image_raw"/>
    <remap from="/ssd_infer/output_topic" to="/ssd_infer/bounding_boxes"/>
  </node>
</launch>
